geomesa-features/geomesa-feature-kryo/pom.xml

```diff
@@ -39,11 +39,11 @@
             <groupId>org.parboiled</groupId>
             <artifactId>parboiled-scala_${scala.binary.version}</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-security_${scala.binary.version}</artifactId>
-            <scope>test</scope>
-        </dependency>
+        <!--<dependency>-->
+            <!--<groupId>org.locationtech.geomesa</groupId>-->
+            <!--<artifactId>geomesa-security_${scala.binary.version}</artifactId>-->
+            <!--<scope>test</scope>-->
+        <!--</dependency>-->
```

Comment:
any reason for removing this? we're going to need it to work with security on the classpath